### ISSUE TITLE
How can i destroy a dialog instance?

### ISSUE THREAD

**Issue by colesnicov** [loadingio/ldcover#8]
Hi. Good work. But how can I destroy the dialog object and then create it again with the same div?

Something like:

```
<div class="ldcv">      <!-- root of the dialog -->
  <div class="base">    <!-- optional. control the position of dialog -->
    <div class="inner"> <!-- dialog body. transition happens here -->
      your content here...
    </div>
  </div>
</div>

var ldcv = new ldCover({root: ".ldcv"});
ldcv.toggle();
```

I would need to call something like

```
ldcv.destroy();
ldcv = null;
```

and somewhere else

```
if(null === ldcv){
  ldcv = new ldCover({root: ".ldcv"});
  ldcv.toggle();
}
```

Thanx.

**Comment by zbryikt** [comment]
hi, I added a destroy function that 
 - toggle off dialog
 - remove eventListeners
 - remove placeholder dom and insert original dom back

You can find it in the feature/destroy branch:
https://github.com/loadingio/ldcover/tree/feature/destroy

please give it a test and any feedback is welcome. :)

**Comment by zbryikt** [comment]
shipped in release 3.4.0

https://github.com/loadingio/ldcover/releases/tag/v3.4.0
